.release-it.json

```diff
@@ -0,0 +1,16 @@
+{
+  "github": {
+    "release": true,
+    "draft": true,
+    "releaseName": "${version}"
+  },
+  "npm": {
+    "publish": false
```

Comment:
Shouldn't it be set to true?

Reply:
Probably 🤔 In Dashboard it's like this. I think they open draft release and a Github action is publishing

In this release, I will set `publish: true` and we will improve

Reply:
Dashboard does not publish packages to npm (its a standalone app)